wiki/Tournaments/RMoT_Invitational/1/en.md

```diff
@@ -0,0 +1,115 @@
+| Host              | ![][flag_SI] [Redavor](https://osu.ppy.sh/u/3328606)|
+| Admins            | ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825), ![][flag_SI] [kristi71111](https://osu.ppy.sh/u/3826105), ![][flag_TR] [oralekin](https://osu.ppy.sh/u/7631823)|
+| Referees          | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Mappool Selectors | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Streamers         | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825)|
+
+## Prizes
+
+The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
+
+### Predictions
+
+Anyone was allowed to predict the outcome of the tournament until the starting date. The predictions took place on Challonge and the top three predicters were awarded half the amount of Points awarded in RMoT's.
+
+## Players
+
+16 players are admitted into each Invitational. Players eligible for Invitational 2 are:
+
+* ![][flag_PL] [Sobe](https://osu.ppy.sh/u/8063872) (formerly known as Sobczyq)
+* ![][flag_NL] [TomBoBotSer](https://osu.ppy.sh/u/7273315)
+* ![][flag_PH] [- Neon -](https://osu.ppy.sh/u/7273455)
+* ![][flag_RU] [nikervm](https://osu.ppy.sh/u/6832092)
+* ![][flag_US] [xdd1](https://osu.ppy.sh/u/7292597)
+* ![][flag_DE] [-Rawn](https://osu.ppy.sh/u/6600592)
+* ![][flag_NL] [Senpaii Henttai](https://osu.ppy.sh/u/7093651)
+* ![][flag_US] [EZdeath](https://osu.ppy.sh/u/7030260)
+* ![][flag_US] [\[Sony\]](https://osu.ppy.sh/u/8750694) (formerly known as sonyguy699)
+* ![][flag_US] [AziiFox](https://osu.ppy.sh/u/8133870) (formerly known as Crystalsim)
+* ![][flag_SE] [ErroTheCube](https://osu.ppy.sh/u/4425262)
+* ![][flag_PL] [CarlosH20](https://osu.ppy.sh/u/8102219)
+* ![][flag_US] [Lifemarr](https://osu.ppy.sh/u/7974462)
+
+**Notice:** ErroTheCube and Senpaii Henttai both reached semi-finals twice or more. Due to this, 3 people who have not reached semi-finals were selected to participate in the tournament, filler players.
+The filler players are chosen according to a few criteria in the following priority:
+
+1. Having participated in and reached QF (quarter-finals) once out of the last 4 RMoT's
+1. Number of times reached QF
+1. Number of times participated in RMoT
+1. Current PP
+
+![][flag_PL] [dark_assasin16](https://osu.ppy.sh/u/6178640), ![][flag_PL] [MagBlood](https://osu.ppy.sh/u/6178640) and ![][flag_PL] [MinatoIDKun](https://osu.ppy.sh/u/6178640) gained the right to participate in the Invitational, MinatoIDKun being seeded lower than the other three.
```

Comment:
So, how are they seeded? The wording need to be fixed but first, need context.

?
?
MinatoIDKun

?